rpc/backend/backend.go

```diff
@@ -197,7 +197,7 @@ func (b *EthermintBackend) PendingTransactions() ([]*rpctypes.Transaction, error
 	}
 
 	transactions := make([]*rpctypes.Transaction, pendingTxs.Count)
```

Comment:
set the initial capacity to 0 and then use append. Otherwise, you might have empty items if the tx is not from the EVM module

Reply:
yeah i remember this being an issue... but i just tested it without append and there seems to be no nil transactions inside the array anymore o.o

Reply:
oh hmm was not aware of this

Reply:
did you try it without EVM transactions, for example a delegation, governance proposal or a bank transfer?

Reply:
i agree this needs to be changed back to append